src/xdocs/checks/annotation/annotationusestyle.xml

```diff
@@ -27,7 +27,7 @@
         </p>
         <p>
           Using the <code>ElementStyleOption.EXPANDED</code> style is more verbose.
-          The expanded version is sometimes referred to as "named parameters" in other languages.
+          The expanded version is sometimes referred to as &quot;named parameters&quot; in other languages.
```

Comment:
Please simply wrap to new line this sentence to make xml comply rule of 100.

Do same for all others.

Reply:
This one yes, because it's a paragraph. But most are examples. I see no point in this when it's a generated file.

Reply:
Oh, yes good point. It will be generated file

Reply:
Please restore suppression.